static/styles.js

```diff
@@ -1,212 +1,183 @@
+    "#192080",
+    "#ffffff",
+    "#111",
+    "#818181",
+];
 
+const lightTheme = [
+    "#c0bfbc",   // Background Color
+    "#000000",   // Chat Color
+    "#000000",   // Message Color
+    "#deddda",   // Chatbox Color
+    "#c0bfbc",   // Sides Color
+    "#000000",   // Sidebar Background Color
+    "#3b3b3b",   // Sidebar Border Color
+    "#ffffff",   // Sidebar Text Color
+    "#5A5A5A",   // Topleft Background Color
+    "#1b0670",   // Topleft Text Color
+    "#3daec4",   // Send Button Background Color
+    "#33575e",   // Send Button Text Color
+    "#b9c6c9",   // Sidenav Color
+    "#192080",   // Sidenav Link Color
+    "#000000",     // Room Text color
+    "#000000",      // Room Line color
+    "#fff"       //Topbar Background Color
+];
 
-function set4thStyle() {
-    // get all id tags
-    let body = document.getElementsByTagName("body")[0]; 
-    let chat = document.getElementById("chat");
-    let message = document.getElementById("message");
-    let chatbox = document.getElementById("chatbox");
-    let sides = document.getElementById("sides");
-    let topleft = document.getElementById("topleft");
-    let send = document.getElementById("send");
-    let sidebar = document.getElementById("activenav");
-    let sidenav = document.getElementsByClassName("sidenav")[0];
-    let snav_text = sidenav.getElementsByTagName("a");
-    let snav_iter = snav_text.length;
-    // then set the color to what it is in the css document
-    body.style.backgroundColor = "#cfcfcf";
-    chat.style.color = "#926f03";
-    message.style.color = "#000000";
-    chatbox.style.backgroundColor = "#d4d4d4";
-    sides.style.backgroundColor = "#0909ff";
-    sidebar.style.backgroundColor ="#0000e6";
-    topleft.style.backgroundColor = "#d4d4d4";
-    topleft.style.color = "#930000";
-    send.style.backgroundColor = "#ff0000";
-    send.style.color = "#ffffff";
-    sidenav.style.backgroundColor = "#550000";
-    // for loop to cycle thru links in sidebar
-    for (let i = 0; i < snav_iter; i++) {
-    snav_text[i].style.color = "#ffffff";
-    }
-}
+const ogDevTheme = [
+    "#000000",  // Background Color
+    "#228e3d",  // Chat Color
+    "#000000",  // Message Color
+    "#181616",  // Chatbox Color
+    "#121212",  // Sides Color
+    "#171717",  // Sidebar Background Color
+    "#3b3b3b",  // Sidebar Border Color
+    "#ffffff",  // Sidebar Text Color
+    "#121212",  // Topleft Background Color
+    "#696969",  // Topleft Text Color
+    "#192080",  // Send Button Background Color
+    "#ffffff",  // Send Button Text Color
+    "#111",     // Sidenav Color   
+    "#818181",  // Sidenav Link Color
+    "#000000",     // Room Text color
+    "#000000"      // Room Line color
+];
 
+const devTheme = [
+    "#000000",
+    "#18691f",
+    "#000000",
+    "#0d0d0d",
+    "#080808",
+    "#080808",
+    "#0d0d0d",   // Sidebar Background Color
+    "#3b3b3b",   // Sidebar Border Color
+    "#ffffff",   // Sidebar Text Color
```

Comment:
for the values that have names in css, maybe we can change them to those names (ex: line 137 would change to `white`)

it's just a suggestion

Reply:
I kept the CSS the same because it all blends in together.

Reply:
understandable